modules/bookList.js

```diff
@@ -0,0 +1,49 @@
+export default class BookList {
+  constructor() {
+    this.bookList = [];
+    this.displayInfo();
+  }
+
+  add(title, author) {
+    if (title && author) {
+      const newBook = {
+        title,
+        author,
+      };
+      this.bookList.push(newBook);
+      localStorage.setItem('bookList', JSON.stringify(this.bookList));
+      // this.bookList = JSON.parse(localStorage.getItem('bookList'));
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.